.github/workflows/e2e-all-k8s.yml

```diff
@@ -19,9 +19,9 @@ jobs:
       fail-fast: false
       matrix:
         globalnet: ['', 'globalnet']
-        # Top/bottom of supported version range. For details:
+        # All non-EOL K8s versions not tested in E2E-full
         # https://submariner.io/development/building-testing/ci-maintenance/
-        k8s_version: ['1.26', '1.27', '1.28', '1.29']
+        k8s_version: ['1.27', '1.28']
```

Comment:
Isn't the purpose of this job to test all K8s versions, specifically the intermediate ones? Otherwise it looks like this job is essentially the same as E2E full.

Reply:
The top/bottom of the range are already tested with this same test matrix in E2E-full. There's no point in testing them twice. This job fills the gaps not covered by E2E-full to provide full K8s version coverage. See the docs update: https://github.com/submariner-io/submariner-website/pull/1114

Reply:
ahh ok  - I missed that.